scm-webapp/src/main/java/sonia/scm/api/v2/resources/TagRootResource.java

```diff
@@ -98,6 +109,53 @@ public Response getAll(@PathParam("namespace") String namespace, @PathParam("nam
     }
   }
 
+  @POST
+  @Path("")
+  @Produces(VndMediaType.TAG_REQUEST)
+  @Operation(summary = "Create tag", description = "Creates a new tag and returns it", tags = "Repository")
+  @ApiResponse(
+    responseCode = "201",
+    description = "create success",
+    headers = @Header(
+      name = "Location",
+      description = "uri to the created tag",
+      schema = @Schema(type = "string")
+    )
+  )
+  @ApiResponse(responseCode = "401", description = "not authenticated / invalid credentials")
+  @ApiResponse(responseCode = "403", description = "not authorized, the current user has no privileges to read the tags")
+  @ApiResponse(
+    responseCode = "404",
+    description = "not found, no tag with the specified name available in the repository",
+    content = @Content(
+      mediaType = VndMediaType.ERROR_TYPE,
+      schema = @Schema(implementation = ErrorDto.class)
+    ))
+  @ApiResponse(
+    responseCode = "500",
+    description = "internal server error",
+    content = @Content(
+      mediaType = VndMediaType.ERROR_TYPE,
+      schema = @Schema(implementation = ErrorDto.class)
+    ))
+  public Response create(@PathParam("namespace") String namespace, @PathParam("name") String name, @Valid TagRequestDto tagRequest) throws IOException {
```

Comment:
We should add an `@Operations` annotation like [this one](https://github.com/scm-manager/scm-manager/blob/develop/scm-webapp/src/main/java/sonia/scm/api/v2/resources/BranchRootResource.java#L214) here